### ISSUE TITLE
Calendar edit enhance

### ISSUE THREAD

**Issue by Freshou** [nextcloud/calendar#5125]
### Is your feature request related to a problem? Please describe.

In the edit calendar dialog, the « delete calendar » button is at the « standard place for a « save button »

### Describe the solution you'd like

Moving the delete calendar button to the left and put a save button on the right… clicking on the X of the dialog, for the end users, doesn’t mean that it will save their change…

and it will prevent user thinking about saving and delete their calendar instead….

**Comment by ChristophWurst** [comment]
:eyes: @nimishavijay @jancborchardt

**Comment by jancborchardt** [comment]
Yeah, sounds reasonable. Delete calendar as well as export are a bit prominent here? How about
- moving delete and export into a 3 dot menu
- reducing the importance of the share button to tertiary
- introducing a Save button on the right (secondary)

Before | After
-|-
![image](https://user-images.githubusercontent.com/925062/229811221-8ab2e266-7612-4d72-818a-b931dbdded41.png)|![image](https://user-images.githubusercontent.com/925062/229811023-639dac61-7599-4ffe-a996-8723104ea8d3.png)

**Comment by ChristophWurst** [comment]
I'm unable to find it but we moved the *Delete* button out of an action menu in the past because it took so many clicks to delete an event.

**Comment by jancborchardt** [comment]
Then we switch the "Copy private link" and the "Delete" function? That is, "Copy private link" goes into the menu with "Export", and "Delete" is a tertiary button between the 3-dot button and "Save"?

**Comment by Freshou** [comment]
Seem good to me !

**Comment by ishaan730** [comment]
Hi @jancborchardt I wanted to share my thoughts on enhancing the calendar edit feature. I've included an optional divider. However, I've also made some significant changes to the private and public links as well as the action buttons. I'd like to explain my rationale behind these changes. I look forward to hearing your thoughts on these changes. 

![Group 140](https://user-images.githubusercontent.com/49994669/232743731-fd9399e1-2d8b-450e-8fc5-8344db4c20f9.jpg)

_**Here’s why we should consider the dropdown option for Public and Private Link.**_

1. Clutter-free design: By placing the public and private options inside a dropdown menu, we can avoid clutter on the modal, making it visually clean and organized. Dropdown menus will also help to group related options together and hide them from view until the user specifically selects the dropdown, reducing visual noise and cognitive load.

2. Clear and concise labeling: Using labels like "Public" and "Private" for the options in the dropdown menu conveys their purpose directly to the user. This clarity in labeling can help users easily understand the functionality of the options, without any ambiguity.

3. Consistency in design: Placing the options inside a dropdown menu can maintain consistency in the design pattern since we have other dropdown menus elsewhere in our application. This will help users build a mental model of how the interface works. 

4. Space-saving: Dropdown menus are an efficient way to save space on the modal as they collapse the options until needed, allowing you to utilize limited screen real estate effectively. This is especially useful in modals where space may be constrained, and you want to keep the design compact.

5. Ease of interaction: Having a "Copy Link" tertiary button positioned nearby the dropdown menu allows for easy interaction. Users can select the desired option from the dropdown menu, and then click the "Copy Link" button to initiate the copy action. This streamlined interaction flow can enhance the usability of the modal, as users can quickly understand and complete the task of copying the desired link

**Why Delete(Secondary) and save(primary) buttons below** – By keeping the "Save" button as the primary action and the "Delete" button as the secondary action can provide a better user experience by preventing accidental deletion, creating a clear hierarchy of actions, reducing cognitive load, maintaining consistency with design patterns, and encouraging positive actions.

Thank you!
